tutorbackup/templates/backup/build/backup/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.8
+ENV PYTHONUNBUFFERED 1
+RUN curl -fsSL https://www.mongodb.org/static/pgp/server-4.2.asc | gpg --dearmor | tee /usr/share/keyrings/mongodb-ce-archive-keyring.gpg && \
+    echo "deb [arch=amd64,arm64 signed-by=/usr/share/keyrings/mongodb-ce-archive-keyring.gpg] https://repo.mongodb.org/apt/debian stretch/mongodb-org/4.2 main" | tee /etc/apt/sources.list.d/mongodb-org-4.2.list && \
```

Comment:
`stretch` strikes me as a little old. Per https://repo.mongodb.org/apt/debian/dists/ there isn't a repo for Debian Bullseye, but you could either install from `buster/mongodb-org/4.2` and leave the `FROM` line as-is, or you could even use `FROM python:3.8-buster` to make sure that the packages match the distro.

Reply:
Yes it's probably better to change to `python:3.8-buster` and use a newer package.